app/services/send_admin_email.rb

```diff
@@ -5,14 +5,30 @@ class SendAdminEmail
   include Service
 
   def call(user:)
+    # returns an array [false, true] or [true, false]
     previous_admin_status = user.previous_changes['admin']
-    send_new_admin_email(user: user) if previous_admin_status[1]
+    send_new_admin_email(user: user)
```

Comment:
I think you might have left this one in by mistake

Reply:
Ah, good catch. Thanks.